### ISSUE TITLE
http://purl.org/spar/cito is broken (404)

### ISSUE THREAD

**Issue by egonw** [SPAROntologies/cito#6]

**Comment by essepuntato** [comment]
Yes, we know. There is a problem, since a few days, of an old purl redirection pointing to sourceforge, and the additional redirection here stopped working as expected. We are trying to fix this at the purl level, and @markjohngraham is already aware of this and up working on it.
